frontend/src/components/Scroll/ChevronScroll.tsx

```diff
@@ -67,23 +64,21 @@ function ChevronScroll(props: React.PropsWithChildren<{}>) {
       <div
         ref={scrollRef}
         onScroll={handleChevronVisibility}
-        className="chevron-scroll-child-container"
+        className="scroll-snap-x mandatory overflow-auto flex items-center gap-4 scrollbar-none"
       >
         {props.children}
       </div>
       <button
         className={combineClasses(
-          "chevron-scroll-right-btn",
-          "align-center",
-          "row",
+          "bg-white px-[32px] py-0 cursor-pointer border-none justify-center",
```

Comment:
> `px-[32px]`

The `chevron-scroll-right-btn` class has a left padding of 15px and a right padding of 32px.

Reply:
Lets also keep the CSS for the above `"align-center"` and `"row"` classes.